app/models/talk.rb

```diff
@@ -356,18 +358,33 @@ def suggestion_summary
   end
 
   def set_kind
-    self.kind =
-      case title
-      when /.*(keynote:|opening\ keynote|closing\ keynote).*/i
-        :keynote
-      when /.*workshop:.*/i
-        :workshop
-      when /.*panel:.*/i
-        :panel
-      when /.*lightning\ talk: .*/i
-        :lightning_talk
-      else
-        :talk
-      end
+    self.kind = case title
+    when /.*(keynote:|opening\ keynote|closing\ keynote|keynote|opening\ keynote|closing\ keynote).*/i
+      :keynote
+    when /.*(lightning\ talk:|lightning\ talk|lightning\ talks).*/i
+      :lightning_talk
+    when /.*(micro\ talk:|micro\ talk).*/i
+      :micro_talk
+    when /.*(panel:|panel).*/i
+      :panel
+    when /.*(workshop:|workshop).*/i
+      :workshop
+    when /.*(gameshow|game\ show|gameshow:|game\ show:).*/i
+      :gameshow
+    when /.*(fishbowl:|fishbowl\ discussion:).*/i
+      :fishbowl
+    when /.*(podcast:|podcast\ recording:|live\ podcast:).*/i
+      :podcast
+    when /.*(q&a|q&a:|ama|q&a\ with|ruby\ committers\ vs\ the\ world|ruby\ committers\ and\ the\ world).*/i
+      :q_and_a
+    when /.*(discussion:|discussion).*/i
+      :discussion
+    when /.*(fireside\ chat:|fireside\ chat).*/i
+      :fireside_chat
```

Comment:
fireside_chat isn't it a kind of q_and_a?

same for panel, fishbowl, discussion aren't they a similar kind at the end but with different naming in conferences?

Reply:
Yeah, they are close to each other, but I guess the formats have different nuances which make them unique.

Reply:
But maybe we can group them somehow that makes sense?

Reply:
I wonder if someone would search specifically for a fishbowl ?

Reply:
I combined them 👍🏼